html-syntax/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <img src="https://cdn.bulbagarden.net/upload/c/cc/065Alakazam.png" alt="Alakazam">
```

Comment:
All content of your page should be either in the `<head>` tag or the `<body>` tag.  The reason it still works is because the browser fixes it for you.

Remember all non visible content of the page should be in the `<head>` tag, and all visible content should be in the `<body>` tag.